core/lib/makeswift/provider.tsx

```diff
@@ -1,19 +1,24 @@
 'use client';
 
-import { ReactRuntimeProvider, RootStyleRegistry } from '@makeswift/runtime/next';
+import { ReactRuntimeProvider, RootStyleRegistry, SiteVersion } from '@makeswift/runtime/next';
 
 import { runtime } from '~/lib/makeswift/runtime';
 import '~/lib/makeswift/components';
 
 export function MakeswiftProvider({
   children,
-  previewMode,
+  siteVersion,
 }: {
   children: React.ReactNode;
-  previewMode: boolean;
+  siteVersion: SiteVersion | null;
 }) {
   return (
-    <ReactRuntimeProvider previewMode={previewMode} runtime={runtime}>
+    <ReactRuntimeProvider
+      apiOrigin={process.env.NEXT_PUBLIC_MAKESWIFT_API_ORIGIN}
+      appOrigin={process.env.NEXT_PUBLIC_MAKESWIFT_APP_ORIGIN}
```

Comment:
Since `apiOrigin` and `appOrigin` are only applicable internally, I wouldn't expose them by default.